.eslintrc.json

```diff
@@ -29,6 +29,7 @@
             "always"
         ],
         "no-unused-vars": 1,
-        "no-console": 0
+        "no-console": 0,
+        "no-var": "error"
```

Comment:
@pksjce I've added this rule to enable `eslint --fix` for it but I agree this should be in a separate PR at least. I'll revert the commit for now and rebase the branch.